src/types/Facility.ts

```diff
@@ -15,3 +18,12 @@ export interface Facility {
   connectionState: 'Not Connected' | 'Connected' | 'Configured';
   timeZone?: string;
 }
+
+export const DEFAULT_STORAGE_LOCATIONS = (): string[] => [
+  strings.FREEZER_1,
+  strings.FREEZER_2,
+  strings.FREEZER_3,
+  strings.REFRIGERATOR_1,
+  strings.REFRIGERATOR_2,
+  strings.REFRIGERATOR_3,
+];
```

Comment:
This list is to render the default list of locations in the 'create new seed bank' flow.